docs/README.rst

```diff
@@ -32,36 +32,67 @@ Overview
 --------
 
 The |cfctlr-long| is a Docker container that runs in a `Cloud Foundry`_ cell.
-It subscribes to the Cloud Foundry NATS message bus and routing API; gathers application route information; and configures the BIG-IP device with a routing policy, emulating the behavior of the Cloud Foundry `Gorouter`_.
+It subscribes to the Cloud Foundry NATS message bus and routing API; gathers application route information (HTTP and TCP); and configures the BIG-IP device with a routing policy, emulating the behavior of the Cloud Foundry `Gorouter`_.
 
-The |cfctlr-long| receives route updates and transforms them into BIG-IP policies when the following events occur in Cloud Foundry:
+The |cfctlr-long| receives route updates and transforms them into BIG-IP policy rules when the following events occur in Cloud Foundry:
 
-- push, delete, and scale applications;
-- create, delete, map, and unmap routes.
+- push, delete, and scale applications (with mapped routes to the application);
+- map, and unmap routes.
 
 For example:
 
-#. Developer pushes fooApp to Cloud Foundry.
+#. Developer pushes fooApp to Cloud Foundry (without --no-route option).
 #. |cfctlr-long| discovers new route information for fooApp.
 #. |cfctlr-long| creates a pool and pool member(s) for each fooApp instance.
 #. |cfctlr-long| updates BIG-IP routing policy directing fooApp requests to the fooApp pool
 #. |cfctlr-long| monitors Cloud Foundry routing table and reconfigures the BIG-IP device when it discovers changes.
 
-The BIG-IP device handles traffic for every Cloud Foundry route and application and load balances to each application instance.
-All route domains are global.
-The |cfctlr-long| can create a total of two (2) BIG-IP virtual servers:
+The BIG-IP device handles traffic for every Cloud Foundry mapped route and load balances to each application instance.
+The |cfctlr| can create a total of two (2) BIG-IP virtual servers for HTTP routes:
 
-- one (1) for http
+- one (1) for http.
```

Comment:
- change '.' to ','
- capitalize http and https

Reply:
Fixed